fuzzing/fuzz-NetDb-HandleNTCP2RouterInfoMsg.cc

```diff
@@ -0,0 +1,30 @@
+
+
+#include <stdint.h>
+#include <stddef.h>
+
+#include <I2NPProtocol.h>
+#include <NetDb.hpp>
+
+#include "fuzzing.h"
+
+
+bool
+fuzzing_testinput(const uint8_t * data, size_t size)
+{
+	i2p::I2NPMessageType msgType;
+
+
+	if(size < 1)
+		return true;
+
+	msgType = (i2p::I2NPMessageType) data[0];
+
+	data++;
```

Comment:
Why this `data++`?

Reply:
It skips over the message type value it took from the fuzzing input.